cfpb/consumerfinance.gov
# Add fallback SVG Icon

Adds a fallback icon when an icon name that is provided, is not valid (not found, invalid SVG).

## How to test this PR

1. Create a page with an icon that doesn't exist.
2. Ensure the missing icon renders out the fallback icon.


## Screenshots
Incoming...
